portfolio/src/main/webapp/comments.html

```diff
@@ -35,24 +35,52 @@ <h1>Comments</h1>
         <h3 class="subtitle">Name:</h3>
         <div class="row">
           <div class="col">
-            <input type="text" class="form-control" placeholder="First name" name="firstName">
+            <input type="text" class="form-control" placeholder="First name" name="firstName" required>
           </div>
           <div class="col">
-            <input type="text" class="form-control" placeholder="Last name" name="lastName">
+            <input type="text" class="form-control" placeholder="Last name" name="lastName" required>
           </div>
         </div>
         <div class="form-group">
           <label for="textarea"><h3 class="subtitle">Leave your comment:</h3></label>
-          <textarea class="form-control" name="textarea" rows="3"></textarea>
+          <textarea class="form-control" name="textarea" rows="3" 
+              placeholder="Leave a comment!" required></textarea>
         </div>
         <button type="submit" class="btn btn-light">Submit</button>
       </form>
       <!-- To see admin page, uncomment next line. -->
       <!-- <a href="/_ah/admin" id="text">View admin page.</a> -->
-      <h3 class="subtitle">Past comments:</h3>
-      <!-- Container used to display the returned list children from showContent() -->
-      <div>
-        <ul id="content-container" class="list-group"></ul>
+      <div class="container">
+        <div class="row">
+          <div class="col">
+            <h3 class="subtitle">Past comments:</h3>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-8">
+            <h5 class="note-btns">Select amount of comments to show:</h5>
+          </div>
+          <div class="col-4">
+            <div class="btn-toolbar" role="toolbar" aria-label="Toolbar with button groups">
+              <div class="btn-group mr-2" role="group" aria-label="First group">
```

Comment:
Is there feedback to the user to let them know which value is currently selected?

Reply:
As of right now, there is no feedback. But I think that would be great value for the user experience. I am thinking of adding a label and generating the text from the JS file but I am going to think about it. For right know, would it be okay to leave it as a TODO?

Reply:
Yeah that would be great! A TODO sounds good here :)